app/src/main/java/mozilla/lockbox/action/RouteAction.kt

```diff
@@ -15,5 +15,6 @@ sealed class RouteAction : Action {
     object SettingList : RouteAction()
     object Back : RouteAction()
     object LockScreen : RouteAction()
+    object Filter : RouteAction()
```

Comment:
Consider a string filter to initialize the list. 
Also: is this the only way we're going to filter this list?

i.e.

```kt
data class TextFilter(text: String? = null) : RouteAction()
```